ui/webui/test/check-language

```diff
@@ -24,7 +24,7 @@ sys.path.append(os.path.join(TEST_DIR, "common"))
 sys.path.append(os.path.join(TEST_DIR, "helpers"))
 sys.path.append(os.path.join(os.path.dirname(TEST_DIR), "bots/machine"))
 
-from installer import Installer
+from helpers.installer import Installer
```

Comment:
Btw. `helpers` don't look like a safe namespace either. I would consider a truly unique namespace or relative imports. Or maybe just prepend these Python paths instead of appending them. Otherwise, this will happen again.

Reply:
Good point, let's fix it for real.

Reply:
I've done some more general re-factoring by moving all the common imports to `anacondalib` (I couldn't make up a better name...) and introduced a new MachineCase subclass which avoids other code duplication.

As a side-effect I now prepend `helpers` so having `python-installer` installed is no longer an issue.